weaknesses/MASVS-RESILIENCE/MASWE-0098.md

```diff
@@ -2,17 +2,43 @@
+
+Malware can exploit Android virtualization software by encapsulating legitimate applications within a virtualized environment, creating seemingly identical user experience. This kind of virtualization allows malicious actors to inject code that bypasses security measures, such as anti-tampering protections, enabling malware to execute fraudulent activities.
+
+Dynamic instrumentation tools, like [GameGuardian](https://gameguardian.net/forum/topic/20322-gameguardian-work-without-root/), can leverage Android virtualization software to operate on non-rooted devices.
+
+## Impact
+
+Maliciously used Android virtualization software can lead to a variety of security issues, including but not limited to:
+
+- **Unauthorized reading of application private data**: Malware utilizing Android virtualization software to virtualize a target application can access its private data. This can lead to exposure of sensitive user information, such as login credentials, financial data, and personal messages, compromising user privacy and security.
+- **Unauthorized alteration of application private data**: By injecting malicious code through virtualization, malware can modify private data within an application. This alteration can result in unauthorized transactions, changes to user settings, or tampering with the application's functionality, potentially causing financial loss or operational disruptions.
+- **Unauthorized escalation of privileges or identity theft**: Virtualization software can create a more permissive environment, enabling the execution of other malicious tools that would normally require root permissions.
+- **Circumventing security mechanisms**: Virtualizers can dynamically tamper with the application's memory without altering the application's integrity. This allows malicious code to bypass certain anti-tampering security measures.
+
+## Modes of Introduction
+
+Virtualization software introduces an additional layer between an application and its underlying operating environment, fundamentally altering how applications interact with system resources. This extra layer creates a more permissive environment, which can have significant implications for both legitimate use and potential threats.
+
+In a virtualized environment, the software effectively isolates the application from the actual operating system and hardware, allowing for a controlled and often more flexible interaction between the application and the system. This permissive environment can bypass standard security measures by providing enhanced privileges and capabilities that would not be available in a non-virtualized setting.
```

Comment:
```suggestion
In a virtualized environment, the software effectively isolates the application from the actual operating system and hardware, allowing the virtualization software to monitor and control any interactions with the system. This is normally not possible on a non-rooted system.
```
The original phrasing is more about 'the app now has more permissions' but I don't see how that's relevant for the security of the application. It's "our" app that we're protecting, so it's not going to start misbehaving now that it suddenly has access to root permissions (e.g).

So this phrasing focusses more on the fact that interaction with the system cannot be trusted.